src/game.h

```diff
@@ -605,6 +605,13 @@ class game
         int get_user_action_counter() const;
 
         signed char temperature;              // The air temperature
+        // Returns temperature modifier from direct heat radiation of nearby sources
+        // @param location Location affected by heat sources
+        // @param direct forces return of heat intensity (and not temperature modifier) of
+        // adjacent hottest heat source
+        int get_heat_radiation( const tripoint &location, bool direct );
```

Comment:
Does not need to be in class game. Only uses the `map m` member of game. Extract from class game and modify declaration to `int get_heat_radiation( const map& m, const tripoint &location, bool direct);`